documentation/book/con-requests-kafka-bridge.adoc

```diff
@@ -34,16 +44,17 @@ m¦Content-Type: application/vnd.kafka.binary.v2+json
 
 |===
 
-You set the embedded data format when creating a consumer using the `consumers/_groupid_` endpoint--for more information, see the next section.
+You set the embedded data format when creating a consumer using the `consumers/groupid` endpoint--for more information, see the next section.
```

Comment:
not having  "_groupid_" but just "groupid", it seems that the endpoint has the fixed "groupid" string into it while it's a placeholder. The endpoints use placeholders with braces, for example `consumers/{groupid}`. Why not using the same in the overall bridge doc?

Reply:
Agreed. I've gone through the Bridge assembly and updated the replaceables to use `{curly braces}`.